query/src/catalogs/impls/catalog/overlaid_catalog.rs

```diff
@@ -233,4 +233,21 @@ impl Catalog for OverlaidCatalog {
             }
         }
     }
+
+    async fn exists_table(
```

Comment:
How these codes is show in this patch, I rember that it's in another patch

Reply:
in the earlier days there's only `exists_database` but no `exists_table` yet, there's another PR working on this? 

if it's true i'd avoid adding exists_table func in this PR to avoid conflicting q.q

Reply:
I've reverted the changes on the catalog trait, plz help review again